src/transport/msg_types/ping.py

```diff
@@ -0,0 +1,31 @@
+import logging
+
+from schema import Schema, And
+from web3 import Web3
+
+from transport.msg_types.base import ADDRESS_REGREX, SignatureSchema
+
+
+logger = logging.getLogger(__name__)
+
+
+PingMessageSchema = Schema({
+    'type': And(str, lambda t: t in ('ping',)),
+    'blockNumber': int,
+    'guardianAddress': And(str, ADDRESS_REGREX),
+    'stakingModuleIds': [int]
+}, ignore_extra_keys=True)
+
+
+def to_check_sum_address(msg: dict):
+    msg['guardianAddress'] = Web3.to_checksum_address(msg['guardianAddress'])
+    return msg
+
+
+UnvetMessageSchema = Schema({
```

Comment:
Should the 'unvet' schema be described here?

Reply:
What do you mean describe?

Reply:
it is ping.py

Reply:
Yeah! Thx